src/stream-tcp.c

```diff
@@ -6034,6 +6034,17 @@ static void StreamTcpPseudoPacketCreateDetectLogFlush(ThreadVars *tv,
     np->flags |= PKT_HAS_FLOW;
     np->flags |= PKT_IGNORE_CHECKSUM;
     np->flags |= PKT_PSEUDO_DETECTLOG_FLUSH;
+    np->livedev = f->livedev;
+
+    if (f->vlan_idx > 0) {
+        np->vlan_id[0] = f->vlan_id[0];
+        np->vlan_idx=1;
```

Comment:
style nitpick: we need spaces before and after the =

Reply:
Changed.